src/relayserver/RelayServer.ts

```diff
@@ -644,17 +664,18 @@ export class RelayServer extends EventEmitter {
     return receipts
   }
 
-  async _isRegistered (): Promise<boolean> {
+  async _getRegistrationBlock (): Promise<number> {
```

Comment:
I was under impression the idea was for the server to send a re-register transaction in case no other transaction was made for a 'registrationBlockRate', so that the server does not become invisible. 
Here it will re-register even if active because it only checks the lates register event.
Is this on purpose?

Reply:
Fixed